src/components/EmailsList.tsx

```diff
@@ -0,0 +1,217 @@
+      { label: "3xl", value: "3xl" },
+      { label: "4xl", value: "4xl" },
+      { label: "5xl", value: "5xl" },
+    ],
+  },
+  fontWeight: {
+    label: "Font Weight",
+    type: "select",
+    options: [
+      { label: "Default", value: "default" },
+      { label: "Thin", value: "thin" },
+      { label: "Extra Light", value: "extralight" },
+      { label: "Light", value: "light" },
+      { label: "Normal", value: "normal" },
+      { label: "Medium", value: "medium" },
+      { label: "Semibold", value: "semibold" },
+      { label: "Bold", value: "bold" },
+      { label: "Extrabold", value: "extrabold" },
+      { label: "Black", value: "black" },
+    ],
+  },
+  padding: {
+    label: "Padding",
+    type: "radio",
+    options: [
+      { label: "None", value: "none" },
+      { label: "Small", value: "small" },
+      { label: "Medium", value: "default" },
+      { label: "Large", value: "large" },
+    ],
+  },
+  color: {
+    label: "Color",
+    type: "select",
+    options: [
+      { label: "Default", value: "default" },
+      { label: "Primary", value: "primary" },
+      { label: "Secondary", value: "secondary" },
+      { label: "Accent", value: "accent" },
+      { label: "Text", value: "text" },
+      { label: "Background", value: "background" },
+    ],
+  },
+  textTransform: {
+    label: "Text Transform",
+    type: "select",
+    options: [
+      { label: "None", value: "none" },
+      { label: "Uppercase", value: "uppercase" },
+      { label: "Lowercase", value: "lowercase" },
+      { label: "Capitalize", value: "capitalize" },
+    ],
+  },
+  includeHyperlink: {
+    label: "Include Hyperlink",
+    type: "radio",
+    options: [
+      { label: "True", value: true },
+      { label: "False", value: false },
+    ],
+  },
+  listLength: {
+    type: "number",
+    label: "List Length",
+    min: 1,
+    max: 100,
+  },
+};
+
+const EmailList: React.FC<EmailListProps> = ({
+  list: emailListField,
+  padding,
+  fontSize,
+  fontWeight,
+  color,
+  textTransform,
+  includeHyperlink,
+  listLength,
+}) => {
+  const document = useDocument();
+  let resolvedEmailList: any = resolveYextEntityField(document, emailListField);
+  if (!resolvedEmailList) {
+    resolvedEmailList = ["sample_email1@gmail.com", "sample_email2@yahoo.com", "sample_email3@msn.com"];
+  } else if (!Array.isArray(resolvedEmailList)) {
+    resolvedEmailList = [resolvedEmailList];
+  }
+  resolvedEmailList.length = Math.min(resolvedEmailList.length, listLength);
+
+  return (
+    <EntityField displayName="Email List" fieldId={emailListField.field}>
+      <ul
+        className={cn(
+          emailListVariants({ padding, fontSize, fontWeight, color, textTransform }),
+          `list-image-[url(${mailIcon})]${includeHyperlink ? " text-blue-600 dark:text-blue-500 hover:underline" : ""}`
+        )}
+      >
+        {resolvedEmailList.map((text: any, index: any) => (
+          <li key={index} className={`mb-2`}>
```

Comment:
Can you try to get it so the email icons and email text are centered with each other?